src/utilcode/clrhost_nodependencies.cpp

```diff
@@ -383,6 +394,10 @@ FastFreeInProcessHeapFunc __ClrFreeInProcessHeap = (FastFreeInProcessHeapFunc) C
 
 const NoThrow nothrow = { 0 };
 
+#ifdef HAS_ADDRESS_SANITIZER
+// use standard heap functions for address santizier
```

Comment:
If we remove the #ifdefs and leave the operators unchanged, there will be duplicate signatures for new, delete, etc during the final link of corerun and other executables as the Clang sanitizer lib defines those.